php-classes/Slate/Term.php

```diff
@@ -190,6 +190,44 @@ public static function getClosestGraduationYear()
         return $Term->getGraduationYear();
     }
 
+    public static function getClosestConcurrentTermIDs($forceRefresh = false)
+    {
+        $cacheKey = "slate-terms/closest-concurrent-ids";
+
+        if (!$forceRefresh && false !== ($termIds = Cache::fetch($cacheKey))) {
+            return $termIds;
+        }
+
+        try {
+            $termIds = static::getClosest()->getConcurrentTermIDs();
+        } catch (TableNotFoundException $e) {
+            $termIds = [];
+        }
+
+        Cache::store($cacheKey, $termIds);
+
+        return $termIds;
+    }
+
+    public static function getClosestMasterConcurrentTermIDs($forceRefresh = false)
+    {
+        $cacheKey = "slate-terms/closest-master-concurrent-ids";
+
+        if (!$forceRefresh && false !== ($termIds = Cache::fetch($cacheKey))) {
+            return $termIds;
+        }
+
+        try {
+            $termIds = static::getClosest()->getMaster()->getConcurrentTermIDs();
```

Comment:
@nbey in this case we want to use `Contained`, as you had noted before:

- `Contained` all terms contained within the given term (e.g., for a year, all the semesters and quarters)
- `Concurrent` all the terms _containing_ the given term (e.g. for a term, the semester and year that are at the same time because they're parent terms)
- `Related` a combination of the two (e.g. for a semester, the year containing it and both quarters within it)

`getClosest()` inherently tries to return a leaf term (which would have no contained terms) and `getClosest()->getMaster()` inherently tries to return a parent-of-all-parents term (which would have no concurrent terms)

Reply:
Ok, @themightychris  it sounds like we should update the method name as well from:
`getClosestMasterConcurrentTermIDs` => `getClosestMasterContainedTermIDs` as well?